messageBundles/gpii-app-qss-settings_en.json

```diff
@@ -59,7 +59,7 @@
         "tooltip": "<p>Changes text size.</p><p>Works with many applications including Edge, Chrome, Firefox, Microsoft Office, and more.</p>",
         "title": "Zoom Top Window",
         "tip": "<p>Zoom Top Window</p><p>Note: Not all windows zoom.</p>",
-        "footerTip": "You can also use the “CTRL” key, plus SCROLL (wheel or gesture) to do Zoom Top Window without using Morphic."
+        "footerTip": "<a href=\"https://www.google.com/url?q=https://docs.google.com/document/d/1vnDDMvQENv3psIvZt_ibtedp4CWm62QnATcjYSxxN00/edit?usp%3Dsharing&sa=D&ust=1562689098659000&usg=AFQjCNE1ldXLzAvn2c41Hsiv5kyYzHyZQw\">Want a faster way to zoom a window?</a>"
```

Comment:
This is a weird reference. Why not just adding this link? https://docs.google.com/document/d/1vnDDMvQENv3psIvZt_ibtedp4CWm62QnATcjYSxxN00

Also, is this supposed to end up in the morphic.world website?

Reply:
Thanks for the observation.
Not sure, I copied it from the specification.

Reply:
For me this link requires a sign-in

Reply:
> For me this link requires a sign-in

That too, that's why I asked if this is supposed to end up in the morphic.world website.
Let's don't merge this until we get a public link for this.